### ISSUE TITLE
Support for NUTS 2024

### ISSUE THREAD

**Issue by raffaem** [ropensci/nuts#3]
Will be support for NUTS 2024 introduced?

**Comment by AAoritz** [comment]
Yes, absolutely! The *JRC* wanted to update the conversion matrices in the next months. As soon as they are out we will integrate them into the package. 

Meanwhile, you can check whether your data is concerned by the changes from 2021 to 2024. Here is an overview:  

![Screenshot from 2024-10-03 17-17-36](https://github.com/user-attachments/assets/0b3cd6e4-1b96-4f95-bfa9-d32c44d37f96)

[Click here](https://ec.europa.eu/eurostat/documents/345175/629341/NUTS2021-NUTS2024.xlsx/2b35915f-9c14-6841-8197-353408c4522d?t=1717505289640) for a an xlsx file with more details from *Eurostat*!

**Comment by AAoritz** [comment]
The 2024 NUTS version is now available in the development version

https://github.com/ropensci/nuts/commit/8637f59e6913ce9d6403a3f96e0c3fdd4b13c8f0

**Comment by raffaem** [comment]
> The 2024 NUTS version is now available in the development version
> 
> [8637f59](https://github.com/ropensci/nuts/commit/8637f59e6913ce9d6403a3f96e0c3fdd4b13c8f0)

Thanks!
